src/main/java/net/aufdemrand/denizen/events/player/PlayerItemTakesDamageScriptEvent.java

```diff
@@ -125,5 +125,6 @@ public void onPlayerItemTakesDamage(PlayerItemDamageEvent event) {
         fire();
         event.setCancelled(cancelled);
         event.setDamage(damage.asInt());
+        event.getPlayer().updateInventory();

```

Comment:
What could the issue be? Cause I've tested it both times, the item still takes damage if the event isn't cancelled. if it is, it doesn't take damage..